kuka_iiqka_eac_driver/src/robot_manager_node.cpp

```diff
@@ -172,78 +158,50 @@ RobotManagerNode::on_cleanup(const rclcpp_lifecycle::State &)
     return FAILURE;
```

Comment:
Cleanup transition prints a lot of warnings
But I think is not the scope of this PR